hashring/hashring.go

```diff
@@ -76,6 +80,7 @@ func New(hashfunc func([]byte) uint32, replicaPoints int) *HashRing {
 		hashfunc: func(str string) int {
 			return int(hashfunc([]byte(str)))
 		},
+		logger: logging.Logger("ring"),
```

Comment:
That's a good question, I did it because this is how it was in node. I think the reason for it is to differentiate between log messages from different nodes, for example, in tickcluster.